foo_ui_columns/status_pane_msgproc.cpp

```diff
@@ -89,23 +89,24 @@ LRESULT status_pane::on_message(HWND wnd,UINT msg,WPARAM wp,LPARAM lp)
             HFONT fnt_old = SelectFont(dc, m_font);
 
             const char * placeholder = "999999999 items selected";
+						const auto default_text_colour = GetSysColor(COLOR_BTNTEXT);
```

Comment:
Looks like some tabs have crept in – this repo uses spaces for indentation so the tabs will need replacing with spaces.